lib/ui/page/home/page/chat/message_field/view.dart

```diff
@@ -422,12 +430,101 @@ class MessageFieldView extends StatelessWidget {
                                   ? null
                                   : const NeverScrollableScrollPhysics(),
                               scrollDirection: Axis.horizontal,
-                              child: Row(
-                                mainAxisSize: MainAxisSize.max,
-                                mainAxisAlignment: MainAxisAlignment.start,
-                                children: c.attachments
-                                    .map((e) => _buildAttachment(context, e, c))
-                                    .toList(),
+                              child: SizedBox(
+                                height: attachmentSize,
+                                child: Row(
+                                  children: [
+                                    SizedBox(
+                                      width:
+                                          (attachmentSize + 4) *
+                                          imgVideosAttachments.length,
+                                      child: ReorderableList(
```

Comment:
Can't we add, please, some kind of UI/UX feedback to the end user when reordering begins? I suggest to perhaps add a small subtle shadow, if possible, when item gets into "reorder mode" (long tap gesture is captured by `ReorderableDelayedDragStartListener`). And a vibration feedback  when reorder starts or happens? We have a helper method to do vibration feedback in `PlatformUtils` variable - please, take a look!